frontend/src/metabase/visualizations/components/ClickActions/ClickActionsPopover.tsx

```diff
@@ -134,7 +134,12 @@ export class ClickActionsPopover extends Component<
         opened={!!popoverAnchor}
         onClose={this.close}
         position="bottom-start"
-        width={700}
+        width="max-content"
+        styles={{
+          dropdown: {
+            maxWidth: 700,
```

Comment:
<img width="1767" alt="image" src="https://github.com/user-attachments/assets/b897cf66-6fa2-4179-99d8-64691811eab6" />
are you sure you're setting the maxWidth on the correct component?
this is me on core app and I don't see any max width being applied

Reply:
I think `ClickActionsPopover` should be used for both the core app and the SDK. I don't think we have a separate component for the SDK. That's weird...

Reply:
Yeah, I'd like to understand why they're behaving differently before we add custom fixes, reading the component code i'd expect to have a maxWidth on core app too but I don't see it working

Reply:
@npretto 
let me convert it to the draft.
I have an assumption that it happens only for Shoppy, maybe because of different Mantine versions?

Reply:
> Yeah, I'd like to understand why they're behaving differently before we add custom fixes, reading the component code i'd expect to have a maxWidth on core app too but I don't see it working

The thing is that for Core app I see that the max-width that is set via code is overridden by Mantine with a value equal to a viewport width. The same behavior happens with `master`
For the Shoppy + SDK I don't see such override by Mantine

Reply:
I can repro it on storybook too on http://localhost:6006/?path=/story/embeddingsdk-interactivedashboard--default

removing `width={700}` seems enough to solve the issue though:
<img width="348" alt="image" src="https://github.com/user-attachments/assets/227d1ab9-1b3c-4c74-b5b2-ebbc936ecad5" />
<img width="812" alt="image" src="https://github.com/user-attachments/assets/2a9e700a-3a68-447e-b46d-ef1b814dd172" />

Reply:
We found a root cause. Waiting confirmation and will update the PR

Reply:
@npretto, so, after all, in this PR, the `width` prop removal is used.

The removal of the `use-popover` override for v53 is a bit tricky because it requires an upgrade of the `mantine` version to `6.0.22`, which also introduces some other changes.

So we will do it for `master` only